tests/Kernel/Paragraphs/MediaParagraphsTest.php

```diff
@@ -688,4 +697,97 @@ public function testBanner(): void {
+    $field_config = $this->container->get('entity_type.manager')->getStorage('field_config')->load('media.iframe.oe_media_iframe');
+    $field_config->set('translatable', TRUE)->save();
+    $this->container->get('router.builder')->rebuild();
+
+    // Create unpublished Iframe media with required fields to check access.
+    $media_storage = $this->container->get('entity_type.manager')->getStorage('media');
+    $media = $media_storage->create([
+      'bundle' => 'iframe',
+      'name' => 'Test Iframe',
+      'oe_media_iframe' => '<iframe src="http://example.com/iframe"></iframe>',
+      'status' => 0,
+    ]);
+    $media->save();
+
+    // Create a paragraph with required fields only.
+    $paragraph = $this->container
+      ->get('entity_type.manager')
+      ->getStorage('paragraph')->create([
+        'type' => 'oe_iframe_media',
+        'field_oe_media' => [
+          'target_id' => $media->id(),
+        ],
+      ]);
+    $paragraph->save();
+
+    // Assert unpublished media isn't shown.
+    $html = $this->renderParagraph($paragraph);
+    $this->assertNotContains('figure', $html);
+    $this->assertNotContains('http://example.com/iframe', $html);
+    $this->assertNotContains('ecl-u-type-heading-2', $html);
+
+    // Publish media.
+    $media->setPublished()->save();
+    // Since static cache is not cleared due to lack of requests in the test we
+    // need to reset manually.
+    $this->container->get('entity_type.manager')->getAccessControlHandler('media')->resetCache();
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+    $iframe = $crawler->filter('figure.ecl-media-container.ecl-media-container--custom-ratio div.ecl-media-container__media.ecl-media-container__media--ratio-custom iframe');
+    $this->assertContains('http://example.com/iframe', $iframe->attr('src'));
+    $this->assertNotContains('ecl-u-type-heading-2', $html);
+
+    // Assert "Full width" field.
+    $paragraph->set('field_oe_iframe_media_full_width', TRUE)->save();
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+    $iframe = $crawler->filter('figure.ecl-media-container.ecl-media-container--fullwidth.ecl-media-container--custom-ratio div.ecl-media-container__media.ecl-media-container__media--ratio-custom iframe');
+    $this->assertContains('http://example.com/iframe', $iframe->attr('src'));
+
+    // Assert ratio.
+    $media->set('oe_media_iframe_ratio', '1_1')->save();
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+    $iframe = $crawler->filter('figure.ecl-media-container.ecl-media-container--fullwidth div.ecl-media-container__media.ecl-media-container__media--ratio-1-1 iframe');
+    $this->assertContains('http://example.com/iframe', $iframe->attr('src'));
+
+    // Assert title and full width.
+    $paragraph->set('field_oe_title', 'Iframe paragraph title');
+    $paragraph->set('field_oe_iframe_media_full_width', FALSE)->save();
```

Comment:
This should be the default behaviour. Not sure why testing it after testing the rendering for full width being enabled.

Reply:
By default it is FALSE and here I just return paragraph to the default state.